src/jit/flowgraph.cpp

```diff
@@ -4857,7 +4859,8 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, BYTE*
                     fgObserveInlineConstants(opcode, pushedStack, isInlining);
                 }
                 break;
-
+            case CEE_RET:
+                retBlocks++;
```

Comment:
Nope, `CEE_JMP` is needed but we also need to look for calls with a tailcall prefix.